todoapp/todos/serializers.py

```
@@ -1,12 +1,14 @@
 from django.contrib.auth import get_user_model
 from rest_framework import serializers
 
-from users.serializers import CustomUserSerializer
 from todos import models as todos_models
+from users import serializers as user_serializers
 
 
 class TodoWithUserSerializer(serializers.ModelSerializer):
-    creator = CustomUserSerializer(read_only=True, source="user", exclude=["id"])
+    creator = user_serializers.CustomUserSerializer(
+        read_only=True, source="user", exclude=["id"]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)